test/integrationTests/testAssets/singleCsproj/.gitignore

```diff
@@ -0,0 +1,2 @@
+**/obj/
```

Comment:
Do we need this?

Reply:
This ignores the obj directory for integrationtests, which I think is fine.

Reply:
Ok. Just merged master. Will merge once travis approves :)